spec/controllers/schedule_spec.rb

```diff
@@ -0,0 +1,154 @@
+require "rails_helper"
+
+RSpec.describe "Schedules", type: :request do
+  describe "GET /index" do
+    before do
+      FactoryBot.create_list(:schedule, 10)
+      @schedule = Schedule.first
+      get "/api/v1/schedules"
+      @first_schedule_data = json["data"][0]["attributes"]
+    end
+    
+    it "returns status code 200 (success)" do
+      expect(response).to have_http_status(:success)
+    end
+
+    it "returns all schedules" do
+      expect(json["data"].size).to eq(10)
+    end
+
+    it "returns correct first_day" do
+      first_day_year = @first_schedule_data["last_day"].split("T").first.split("-").first.to_i
+      first_day_month = @first_schedule_data["last_day"].split("T").first.split("-").second.to_i
+      first_day_day = @first_schedule_data["last_day"].split("T").first.split("-").third.to_i
+      expect(Date.new(first_day_year, first_day_month, first_day_day)).to eq(@schedule.last_day.to_date)
```

Comment:
I believe this could be simplified by calling `expect(@first_schedule_data["last_day"].to_datetime).to eq(@schedule.last_day)`.  If so, the other tests could also be consolidated.